### DISCUSSION TITLE
Running a quad with a servo on a spare S pad

### DISCUSSION THREAD

**Discussion by duncansshand** [iNavFlight/inav#6554] [Q&A] [answered]
Currently running a HGLRC F722 Zeus with iNav 2.6 and 4 motors on S1,S2,S3 and S4.

If I was to add a servo in the iNav mixer to say S7 and then connect up the servo to the 5v, ground and S7 pad on the FC do you think this would work?

I could assign the servo in the mixer to a switch on my RC radio, I think this would work but I'm sure someone who knows more could correct me.

Thanks

Duncan

**Comment by DzikuVx** [comment, marked as answer]
This board allows servos in a Multirotor mode on S7 and S8. Yes, it will work with a function you will assign to it
